spec/shared_contexts/l10n.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+RSpec.shared_context('l10n') do |locale|
+  before :all do
+    @old_locale = Locale.current
+    Locale.current = locale
+    Puppet::GettextConfig.setup_locale
+
+    # overwrite stubs with real implementation
+    ::Object.send(:remove_method, :_)
+    ::Object.send(:remove_method, :n_)
+    class ::Object
+      include FastGettext::Translation
+    end
```

Comment:
This is kind of gross, open to suggestions about how to improve

Reply:
Hmm, maybe `Object.extend(FastGettext::Translation)` will overwrite the existing methods from stubs?

Reply:
thoughts @Magisus?

Reply:
Looks like this is copied from what we're doing in [real code](https://github.com/puppetlabs/puppet/blob/846e5e41d50d363e738658588c497313a59900a5/lib/puppet/gettext/config.rb#L18-L21) here in Puppet; in the `gettext-setup` gem we do bascially this, just written differently: https://github.com/puppetlabs/gettext-setup-gem/blob/master/lib/gettext-setup/gettext_setup.rb#L28

Possibly a slight preference for staying consistent with what we're doing in live code here, just for consistency.

Reply:
Sorry @Magisus, just to confirm you’re ok with how it is in the PR as it matches what we’re doing in puppet’s gettext config?